examples/service/stateful/providers/azure/create/main.go

```diff
@@ -121,6 +121,7 @@ func main() {
 							TagValue: spotinst.String("Tamiry@netapp.com"),
 						},
 					},
+					UserData: spotinst.String("VGhpcyBpcyBhIHZhbGlkIGJhc2UgNjQgdXNlciBkYXRhIHNjcmlwdA=="),
```

Comment:
update as userData instead of actual value

Reply:
done